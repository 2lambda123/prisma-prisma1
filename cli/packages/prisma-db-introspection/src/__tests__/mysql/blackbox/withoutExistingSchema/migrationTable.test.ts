import testSchema from '../common'

describe('Reserved Tables', () => {
  test('ignore relation table', async () => {
    await testSchema(`CREATE TABLE \`Test\` (
      \`pk\` varchar(55) NOT NULL,
      \`a\` char(1) DEFAULT NULL UNIQUE
      );
      
      CREATE TABLE \`_Migration\` (
        \`pk\` int NOT NULL PRIMARY KEY
      );`)
  })
})
