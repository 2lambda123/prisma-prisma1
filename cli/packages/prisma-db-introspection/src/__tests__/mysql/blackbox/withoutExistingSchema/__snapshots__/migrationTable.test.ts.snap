// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reserved Tables ignore relation table 1`] = `
"type Test {
  id: ID @pgColumn(name: \\"a\\") @unique
  pk: String!
}"
`;

exports[`Reserved Tables ignore relation table 2`] = `
"type Test {
  a: ID @id
  pk: String!
}"
`;
